### DISCUSSION TITLE
Failed to pull image for kubelet from ghcr.io

### DISCUSSION THREAD

**Discussion by peterkuiper** [siderolabs/talos#5180] [Q&A] [answered]
I am following [this](https://www.talos.dev/v0.14/single-board-computers/rpi_4/) guide but am running into the following issue.

I'm running talosctl from another rpi4:

```
curl -Lo /usr/local/bin/talosctl https://github.com/talos-systems/talos/releases/latest/download/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-armv7
chmod +x /usr/local/bin/talosctl
talosctl apply-config --insecure --interactive --nodes 10.255.250.157
```
While bootstrapping, I'm getting the following error:

```
retrying error, failed to pull image "ghcr.io/talos-systems/kubelet:v1.23.1": failed to resolve reference "ghcr.io/talos-systems/kubelet:v1.23.1": failed to authorize: failed to fetch anonymous token: unexpected status: 401 Unauthorized
```

The bootstrap is now stuck retrying to pull the above image and logging a `controller failed` error message:

```
[talos] controller failed {"component": "controller-runtime", "controller": "k8s.ManifestApplyController", "error": "error creating mapping for object /v1/Secret/bootstrap-token-8esmZo": Get \"https://localhost:6443/api?timeout=32s\": dial tcp [::1]:6443: connection refused"}
```

Any ideas?

**Comment by smira** [comment, marked as answer]
There's an issue with GitHub organization migration on our side. Once it is
fixed, the image will be available once again.

Sorry for the inconvenience.

**Reply by peterkuiper** [reply]
Thanks for the update!

**Reply by smira** [reply]
It should be fixed now!

**Reply by peterkuiper** [reply]
Looks like it won't fix itself:

```
service[kubelet](Waiting): Error running Containerd (kubelet), going to restart forever: task "kubelet" failed: exit code 1
```

I'm guessing there is nothing I can do to troubleshoot this?

I will retry tomorrow with a fresh install :)

**Reply by smira** [reply]
I think that is something different, not related to the org rename as the image was pulled

**Reply by peterkuiper** [reply]
I was able to this running after re-flashing the sdcard! Thanks @smira!
